bin/check-process.rb

```diff
@@ -176,8 +181,10 @@ class CheckProcess < Sensu::Plugin::Check::CLI
   def read_pid(path)
     if File.exist?(path)
       File.read(path).strip.to_i
-    else
+    elsif config[:file_pid_crit].nil?
```

Comment:
shouldn't this be `!config[:file_pid_crit].nil?` as you are wanting to crit when its specified which without I believe would be nil.